### PR TITLE
Upgrade to ocamlformat 0.26.0

### PR DESCRIPTION
**Pull request by Julow**
The aim of this commit is to gather feedback.

Changelog can be found here: https://github.com/ocaml-ppx/ocamlformat/blob/main/CHANGES.md

### CODE REVIEW

File: lib/functoria/key.mli [hunk 1]
```diff
@@ -182,7 +182,7 @@ val v : 'a key -> t
 (** [v k] is the [k] with its type hidden. *)
 
 val abstract : 'a key -> t
-  [@@ocaml.deprecated "Use Functoria.Key.v."]
```
Comment (samoht) [inline lib/functoria/key.mli:185]: @Julow Is this expected? The annotation is supposed to be attached to the top-level item, right?

Comment (Julow) [reply lib/functoria/key.mli:185]: The attribute is unindented but still attached to the same item, yes.

Comment (samoht) [reply lib/functoria/key.mli:185]: what's the rationale for changing this?

Comment (Julow) [reply lib/functoria/key.mli:185]: This is changed here: https://github.com/ocaml-ppx/ocamlformat/pull/2317

It was initially implemented for the Janestreet profile and I thought less indentation is generally better so I changed the default profile too. Do you prefer the previous formatting ?

Comment (samoht) [reply lib/functoria/key.mli:185]: Ha ok, that matches ocp-indent behavior then? If yes, I'm fine with the changes, that's a good enough rational.

Comment (Julow) [reply lib/functoria/key.mli:185]: It does. The main reason was that less indentation is generally better and this is useless indentation.

Comment (samoht) [reply lib/functoria/key.mli]: Was it reverted in the new version you just pushed?

Comment (Julow) [reply lib/functoria/key.mli]: No, the attributes have been removed in cf46b243318265024c88ceb86dc47ca16c3087cc and I rebased this PR on top.

**Comment by Julow** [comment]
OCamlformat 0.26.0 has been released! This PR can be merged (not squashed due to `.git-blame-ignore-revs`) or closed if it's a bit too early to upgrade.

**Review by samoht — APPROVED** [review]

**Comment by samoht** [comment]
Thanks, I'll merge once `ocaml-ci` is green (hopefully soon)

**Pull request merged**